proto/buf.gen.gogo.yaml

```diff
@@ -2,7 +2,7 @@ version: v1
 plugins:
   - name: gocosmos
     out: ..
-    opt: plugins=grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types
+    opt: plugins=grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/gogoproto/types/any
```

Comment:
we are going to need to add upgrade documentation for chains who use `buf` for proto generation

see [example](https://github.com/cosmos/gaia/blob/b53ebdf7e696bb2ccdc7cbafd3623cbb46e81549/proto/buf.gen.gogo.yaml#L5)

Reply:
https://github.com/cosmos/cosmos-sdk/blob/main/UPGRADING.md
https://github.com/cosmos/cosmos-sdk/blob/main/UPGRADE_GUIDE.md

We need entries in both of these files to document this

Reply:
I can add something, but I think because we have the type annotations in place it will work fine as is. The only reason for a change is if someone wants to avoid a direct SDK dependency, like we do with depinject

Reply:
oh i see - yeah you're right this should be fine